stripe-ui-core/src/main/java/com/stripe/android/uicore/text/Html.kt

```diff
@@ -214,8 +214,8 @@ private fun ClickableText(
     text: AnnotatedString,
     modifier: Modifier = Modifier,
     inlineContent: Map<String, InlineTextContent> = mapOf(),
-    color: Color = MaterialTheme.paymentsColors.subtitle,
-    style: TextStyle = MaterialTheme.typography.body2,
+    color: Color,
+    style: TextStyle,
```

Comment:
Can you move these up, before modifier since they don't have a default value anymore? Or set the same default value as above